lxd/main.go

```diff
@@ -45,7 +45,7 @@ func main() {
 // Index of SubCommand functions by command line name
 //
 // "forkputfile", "forkgetfile", "forkmount" and "forkumount" are handled specially in main_nsexec.go
-// "forkgetnet" is partially handled in nsexec.go (setns)
+// "forkgetnet" is partially handled in main_nsexec.go (setns)
```

Comment:
Good catch, can you move that to its own commit so I can include it in stable-2.0 too?

Reply:
Yeah I can make a separate commit and PR for this.